packages/functionals/botpress-hitl/src/db.js

```diff
@@ -9,6 +9,16 @@ function initialize() {
     throw new Error('you must initialize the database before')
   }
 
+  const table = 'hitl_messages'
+  const migrateTable = () =>
+    knex.schema.hasTable(table).then(exist => {
+      if (exist) {
+        knex.schema.alterTable(table, t => {
```

Comment:
I think you need `return` here to ensure the engine awaits on this change

Reply:
@emirotin fixed